src/states/node.rs

```diff
@@ -103,6 +104,8 @@ pub struct Node {
     tick_timer_token: u64,
     timer: Timer,
     tunnels: Tunnels,
+    /// Key is (dst, sections), value is BTreeSet<src>
+    out_of_seq_own_merge: LruCache<(Prefix<XorName>, SectionMap), BTreeSet<Prefix<XorName>>>,
```

Comment:
Isn't that just because there are not many sections in the test?
If there are four balanced sections and `00` and `01` are merging, the section map sent by `01` will include `11` but not `10`, and the one sent by `00` will include `10` but not `11`, won't it?
And even _if_ they include the same sections, one half might have already handled some added or lost node, and the other might not, so the members list can disagree?